src/components/Cart/cart.js

```diff
@@ -0,0 +1,76 @@
+import "../../styles.css";
+import { useCartAndWishListContext } from "../../contexts/cart-wishlist-context";
+
+export default function ({ cartItems }) {
+  const { state, dispatch  } = useCartAndWishListContext();
+
+  return (
+    <div>
+      {" "}
+      {state.itemsInCartArray.map((item) => (
+        <div
+          style={{
+            display: "flex",
+            height: "auto",
+            width: "20rem",
+            margin: "auto",
+            marginTop: "4rem",
+            backgroundColor: "#F3F4F6",
+            padding: "1rem",
+            position: "relative"
+          }}
+        >
+          <button
+            className="close"
+            style={{
+              position: "absolute",
+              marginTop: "-0.25rem",
+              marginLeft: "19.5rem"
+            }}
+            onClick={() => dispatch({type: "REMOVE_FROM_CART", payload: item})}
+          >
+            x
+          </button>
+
+          <img
+            className="responsive-image"
+            src={item.image}
+            style={{ height: "8rem", width: "auto", marginRight: "2rem" }}
```

Comment:
Here also the style tag could be removed. Actually this helps in organizing your code. The code will remain in one file while the styles would be in another file